cmake/QuantLibConfig.cmake.in

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/QuantLibTargets.cmake")
```

Comment:
I'm guessing this file need to be included in the release?  If so, it should go in the `disthook` in the `Makefile.am` in the root.

Reply:
Actually this file is auto-generated from the following command:

```cmake
export(EXPORT QuantLibTargets
    FILE "${CMAKE_BINARY_DIR}/cmake/QuantLibTargets.cmake"
    NAMESPACE QuantLib::
)
```

Not sure it's worth figuring out a way to auto-generate this file for the automake builds, as people using automake probably don't care about the CMake targets. What do you think?

Reply:
No, sorry, I meant this very file — `cmake/QuantLibConfig.cmake.in`.

Reply:
Yes you're right, looks like I should add `cmake/QuantLibConfig.cmake.in` to the `dist-hook`.

However now `check_filelists.sh` is complaining about `ql/CMakeLists.txt contains extra file ql/QuantLibTargets.c`. Should I just filter this specific file out in `check_filelists.sh` since it is auto-generated?

Reply:
I think you can change line 22 in `tools/check_filelists.sh` so it doesn't look for `.c` or `.h` files anymore, same as you did in the rest of the script.

Reply:
Done